recipes/python/python.inc

```diff
@@ -1,49 +1,250 @@
-# -*- mode:python; -*-
-DESCRIPTION = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
-HOMEPAGE = "https://www.python.org/"
```

Comment:
Bah, I expect people to able to do at least a google for 'python'.

Anyways, I added it back